import_export/tmp_storages.py

```diff
@@ -69,23 +70,50 @@ def remove(self):
 
 class MediaStorage(BaseStorage):
     MEDIA_FOLDER = "django-import-export"
+    _storage = None
 
     def __init__(self, **kwargs):
+        super().__init__(**kwargs)
+        if django.VERSION[:2] > (4, 2):
+            from django.core.files.storage import StorageHandler
+
+            sh = StorageHandler()
+            if "import_export" in sh.backends:
```

Comment:
A very minor quibble here but this doesn't feel quite as pythonic as 
```
self._storage = sh.backends.get("import_export") or sh.backends.get("default")
```

Or something like that. Might some functional reason I'm missing for the key based access and if check pattern though.

Reply:
Good call.  I thought there would be a cleaner way of doing this, but it doesn't work like this because of StorageHandler implementing [`__get_item__()`](https://github.com/django/django/blob/a93375e8ab26e7cb22c6ee7e4bf23bf2d6ed170d/django/core/files/storage/handler.py#L24).
Therefore I have to leave it as is.